app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/FractionInteractionViewModel.kt

```diff
@@ -56,19 +57,24 @@ class FractionInteractionViewModel(
     return userAnswerBuilder.build()
   }
 
+  override fun setPendingAnswer(userAnswer: UserAnswer) {
+    answerText.set(userAnswer.plainAnswer)
```

Comment:
@BenHenning Here I'm unable to update the View using this method is there any other way to do this?

Reply:
Can you be specific on what isn’t working? It’s sometimes hard to tell from just the code. The more specific you can be the better.

Reply:
Here I'm able to send the userAnswer (pending answer after configuration change) to the FractionInteractionViewModel but i'm unable to update it using the ObservableField for `answerText` (updated it string -> ObservableField to update the Input through viewmodel). Here I'm able to get the value but I'm unable to update the FractionInteractionView. 

Is the implementation correct 
Is there any other way to update the FractionInteractionView using the ViewModel

Reply:
Hmm. Just to check: did you confirm:
- setPendingAnswer is being called here
- setPendingAnswer is being passed new data
- The current value of answerText when setPendingAnswer is called isn't what you expect (e.g. empty)

If all are true, I'm not exactly sure what's wrong. observable fields should work out of the box, but you could try calling ``notifyChange`` on the view model (per https://github.com/oppia/oppia-android/blob/develop/app/src/main/java/org/oppia/android/app/viewmodel/ObservableViewModel.kt#L23). FWIW I think you can actually avoid observable field if you use notifyChange, but maybe start with both to reduce the number of things that could be wrong here.

Reply:
I checked this:
`answerText` is already set to the value which is coming from `setPendingAnswer` but the FractionInteractionView is not displaying it. 
Also, `notifyChange()` did not work.

Reply:
Hmm. I'm wondering if this is behaving this way because we have a text watcher set. It might be the case that we aren't actually copying over the value to answerText directly from databinding (& instead rely on the text watcher). I won't have time to dig into this right now, but @FareesHussain I suggest searching for whether a text watcher can interfere with databinding.